themes/clean-detailed.omp.json

```diff
@@ -33,15 +33,14 @@
           "foreground": "#ffffff",
           "leading_diamond": "\ue0b2",
           "style": "diamond",
-          "template": "\uf85a CPU: {{ round .PhysicalPercentUsed .Precision }}% | ",
```

Comment:
Copied `{{ round .PhysicalPercentUsed .Precision }}%` in front of MEM: and deleted this line